app/timetables/tests/test_models.py

```diff
@@ -235,3 +237,49 @@ def test_zero_cycle_day_value_cannot_be_saved(self):
             timetable=self.timetable_object)
 
         self.assertRaises(ValidationError, menu_item_three.save)
+
+
+class EventTest(TestCase):
```

Comment:
I see we are testing for failure but aren't we going to test for success? Checking that an event would successfully save